docs/concepts/service-catalog/index.md

```diff
@@ -62,7 +62,7 @@ Service Catalog supports these methods of authentication:
 
```

Comment:
"Similar to ClusterServiceClass, when a new ClusterServiceBroker is added to the cluster, **the Service Catalog** creates a new ClusterServicePlan resource corresponding to each Service Plan available for each Managed Service."

If we are using Service Catalog as a proper name (and I think we are), then "the Service Catalog" should be "Service Catalog".

Reply:
done
yeah, "Service ServiceInstance" was a typo